geometry_msgs/Point center

geometry_msgs/Vector3 half_size